image/rhel/Dockerfile.envsubst

```
@@ -43,7 +44,7 @@ RUN ln -s entrypoint-wrapper.sh /stackrox/admission-control && \
     ln -s entrypoint-wrapper.sh /stackrox/compliance && \
     ln -s entrypoint-wrapper.sh /stackrox/kubernetes-sensor && \
     ln -s entrypoint-wrapper.sh /stackrox/sensor-upgrader && \
-    ln -s /assets/downloads/cli/roxctl-linux-amd64 /stackrox/roxctl && \
+    ln -s /assets/downloads/cli/roxctl-linux-${TARGET_ARCH} /stackrox/roxctl && \
```

What's wrong with
```suggestion
    ln -s /assets/downloads/cli/roxctl-linux-$(uname -m) /stackrox/roxctl && \
```
? Why do you need an explicit build arg?

---

Because uname -m ends up returning aarch64 when inside the Dockerfile :(